export declare function caughtError(e: unknown): Error;
